client/src/components/Validate.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import styles from './ExplorerPreview.module.css'
+import { Button } from './Button'
+import { ReactComponent as PlusIcon } from '../icons/plus.svg'
+import { PreviewIcon } from './PreviewIcon'
+
+//so far added changes in header with connection to api we can add changes here
+
+//left this explorer preview as we might want to show it same way
+
+export const ExplorerPreview = ({

```

Comment:
Shouldn't highlighting be contained in `Carousel `instead of `ExplorerPreview`?
`ExplorerPreview `is just the file explorer in the MyDocuments section. `Carousel` instead is the actual list of text boxes in the Editor.